opentelekomcloud/services/iam/data_source_opentelekomcloud_identity_project_v3.go

```diff
@@ -106,10 +114,18 @@ func dataSourceIdentityProjectV3Read(_ context.Context, d *schema.ResourceData,
 			"Please change your search criteria and try again.")
 	}
 
+	var NewFilteredProjects []projects.Project
```

Comment:
This variable can be placed inside of `if`, also variables inside on functions named starting with lowercase letter